libs/blocks/merch-card/merch-card.js

```diff
@@ -102,6 +102,14 @@ function addMerchCardGridsIfMissing(section) {
   }
 }
 
+const decorateMerchCardLinkAnalytics = (el) => {
+  [...el.querySelectorAll('a')].forEach((link, index) => {
+    const heading = el.querySelector('h3');
+    const analyticsString = `${processTrackingLabels(link.textContent)}-${index + 1}|${heading?.textContent || ''}`;
```

Comment:
@Axelcureno  I found an NPE with this line an fixed it here:https://github.com/adobecom/milo/blob/commerce-v3/libs/blocks/merch-card/merch-card.js#L190C1-L197C1
 can you review if the fix makes sense?

Reply:
Yes, it does. I've applied the fix.